### PR TITLE
build(deps-dev): Bump typescript from 3.5.3 to 3.7.2

### PR DESCRIPTION
**Pull request by kmdrGroch**

### CODE REVIEW

**Comment by kmdrGroch** [comment]
this should push further development. I think we should think of removing some 'any' for more precise types

File: packages/core/src/input/InputFileCollection.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ export default class InputFileCollection {
         You can configure the \`mutate\` property in your stryker.conf.js file (or use \`--mutate\` via command line).`)
         );
       }
-      if (log.isDebugEnabled) {
+      if (log.isDebugEnabled()) {
```
Comment (simondel) [inline packages/core/src/input/InputFileCollection.ts]: Nice find ♥️

**Comment by simondel** [comment]
E2E inconsistency two times in a row... good enough :(

File: packages/core/src/input/InputFileCollection.ts (hunk 1 above)
Comment (kmdrGroch) [reply packages/core/src/input/InputFileCollection.ts]: Typescript found it O.o

Comment (nicojs) [reply packages/core/src/input/InputFileCollection.ts]: superawesome! I love TypeScript

**Pull request merged**